### PR TITLE
Re-enabled additional tests

### PR DESCRIPTION
**Pull request by dreab8**

### CODE REVIEW

**Comment by beikov** [comment]
Same problem again with the old Oracle version that some identifiers are too long in `org.hibernate.userguide.mapping.embeddable.EmbeddableOverrideTest` :)

**Comment by dreab8** [comment]
Hi @beikov I'm going to fix it, thanks.
What about https://github.com/dreab8/hibernate-orm/commit/18213bdb64c5c46f947024879ff71baf103b7e51 ? not sure about it

**Comment by jrenaat** [comment]
Resolved the conflict with the TransactionTimeoutTest due to myPR; kept some of your things in it.

File: hibernate-core/src/main/java/org/hibernate/query/results/DomainResultCreationStateImpl.java [hunk 1]
```diff
@@ -266,7 +268,8 @@ public Expression resolveSqlExpression(
 		else if ( created instanceof ColumnReference ) {
 			final ColumnReference columnReference = (ColumnReference) created;
 			final String columnExpression = columnReference.getColumnExpression();
-			final int jdbcPosition = jdbcResultsMetadata.resolveColumnPosition( columnExpression );
+
+			final int jdbcPosition = jdbcResultsMetadata.resolveColumnPosition( currentResultBuilder.getColumnAlias( columnExpression ) );
```
Comment (beikov) [inline hibernate-core/src/main/java/org/hibernate/query/results/DomainResultCreationStateImpl.java:272]: This and all the related changes shouldn't be necessary. What does that fix?

Comment (dreab8) [reply hibernate-core/src/main/java/org/hibernate/query/results/DomainResultCreationStateImpl.java]: reverted thanks

**Pull request closed without merging**